### PR TITLE
Checked for undefined and blank spaces in label

### PR DESCRIPTION
**Pull request by nishant8BITS**
Added check for undefined and blank spaces at in string (beginning and end) in classification label

### CODE REVIEW

File: lib/natural/classifiers/classifier.js [hunk 1]
```diff
@@ -34,6 +34,15 @@ var Classifier = function(classifier, stemmer) {
 };
 
 function addDocument(text, classification) {
+    
+    // Ignore further processing if classification is undefined 
+    if(typeof classification === 'undefined') return;
+    
+    // If classification is type of string then make sure it's dosen't have blank space at both end  
+    if(typeof classification === 'sytring'){
```
Comment (SuperSpyTX) [inline lib/natural/classifiers/classifier.js]: did you mean 'string' not 'sytring'?

Comment (nishant8BITS) [inline lib/natural/classifiers/classifier.js]: Yes Sorry "sytring" was type it should be string :)

Comment (SuperSpyTX) [inline lib/natural/classifiers/classifier.js]: It is ok.  I'm not a contributor, just like to lurk around possible projects that I like, perhaps might use for a future project of mine.

**Comment by kkoch986** [comment]
thanks!

**Pull request merged**